src/coreclr/inc/clrtypes.h

```diff
@@ -338,7 +338,7 @@ inline UINT64 AlignUp(UINT64 value, UINT alignment)
     return (value+alignment-1)&~(UINT64)(alignment-1);
 }
 
-#ifdef __APPLE__
+#if defined(__APPLE__) || defined(__wasm__)
```

Comment:
It is not clear to me why we need this for wasm when it was used on apple only before.